numpy/ma/core.py

```diff
@@ -8176,8 +8176,13 @@ def round_(a, decimals=0, out=None):
         return out
 
 
-round = round_
```

Comment:
On line 8123 above you need to change the function name from `round_` to `round`